src/index.ts

```diff
@@ -80,6 +80,9 @@ export default class OASNormalize {
         }
 
         const contents = fs.readFileSync(this.file).toString();
+        if (!contents) {
```

Comment:
Might want to also trim any whitespace just in case.

```suggestion
        if (!contents.trim()) {
```

Reply:
good call!